tests/format/js/module-string-names/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`module-string-names-export.js [__babel_estree] format 1`] = `"Cannot read properties of undefined (reading 'raw')"`;
```

Comment:
It seems different Node.js throws different message, let's wait for that babel issue to be fixed then.

Reply:
Fixed by https://github.com/prettier/prettier/pull/11973